Cargo.toml

```diff
@@ -16,7 +16,7 @@ exclude = ["/bors.toml"]
 
 edition = "2018"
 
-rust-version = "1.36.0"
+rust-version = "1.43.1"
```

Comment:
Can you mention that we're bumping MSRV in the commit message? I just want to make sure I don't miss it when I'm updating our CHANGELOG.

Reply:
yeah no worries. How do you want me to word it? Given that this commit doesn't actually change the MSRV, it just documents it correctly (ie. it would not have compiled with the 1.36.0 toolchain even before this change)

Reply:
There is also the module documentation of "lib.rs" that mention the MSRV to update.

Reply:
done. Updated the docs in a new commit. Do you want to do a squash merge and choose your own commit message?

Reply:
Heh, I've apparently turned off squash merging. I'll need to fiddle with that.

In the meantime, could you set the commit message to "Document and test bumped MSRV."